java-checks/pom.xml

```diff
@@ -184,11 +184,6 @@
                   <type>jar</type>
                 </artifactItem>
                 <artifactItem>
-                  <groupId>javax</groupId>
```

Comment:
It's a duplicate of line [#139](https://github.com/SonarSource/sonar-java/pull/943/files/9791754d17d30557a03aa303e233cda221181131#diff-598a04ed7e994c371420982332fc73caR139), that I discovered while I wanted to add the jar containing `@Inject` annotation.